<?xml version="1.0" encoding="utf-8"?>

<launch>
  <node name="serket_thread" pkg="serket_ros" type="serket_thread.py" cwd="node" output="screen"/>
</launch>